text/3516-public-private-dependencies.md

```diff
@@ -229,7 +229,7 @@ In the case where you depend on `foo = "300"`, there isn't a way to clarify that
 
 This doesn't cover the case where a dependency is public only if a feature is enabled.
 
-The warning is emitted even when a `pub` item isn't accessible.
+The warning/error is emitted even when a `pub` item isn't accessible.
```

Comment:
This would be less painful if inaccessible `pub` items were less commonplace. There's already the allow-by-default `unreachable_pub` lint that people can opt into to find and remove inaccessible `pub`s, but it can be needlessly noisy: https://github.com/rust-lang/rust/issues/112615

Reply:
Note that we at least avoid his for bins, examples, etc because we won't tell rustc that any of the dependencies are private.  I've added a note about that to this section (its already noted in the Reference section).

Reply:
The other thing about inaccessible pub is that you have the sealed trait trick which (in some cases) will pull in seemingly-public deps as private because it's a sealed trait.

Probably not a big deal, but worth mentioning that this one use case does  exist.  At least with Rust as it is now, I have yet to find a workaround for sealed traits in the cases where they're applicable.

Reply:
Well, #3323 was merged, right?